### PR TITLE
fzf-git-sh: 2022-09-30 -> 2024-03-17

### PR DESCRIPTION
**Pull request by becknik**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/shells/fzf-git-sh/default.nix [hunk 1]
```diff
@@ -17,13 +17,13 @@
 
 stdenv.mkDerivation rec {
   pname = "fzf-git-sh";
-  version = "unstable-2022-09-30";
+  version = "unstable-2024-02-17";
```
Comment (kirillrdy) [inline pkgs/shells/fzf-git-sh/default.nix:20]:
```suggestion
  version = "0-unstable-2024-02-17";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

File: pkgs/shells/fzf-git-sh/default.nix [hunk 2]
```diff
@@ -17,13 +17,13 @@
...
 
   src = fetchFromGitHub {
     owner = "junegunn";
     repo = "fzf-git.sh";
-    rev = "9190e1bf7273d85f435fa759a5c3b20e588e9f7e";
-    sha256 = "sha256-2CGjk1oTXip+eAJMuOk/X3e2KTwfwzcKTcGToA2xPd4=";
+    rev = "2b93e957684f7daca8b28cb74c9a7e7fc606e81e";
+    sha256 = "sha256-0KFxwx6wGaHXIK0FrAVkodWDMjc9zQjEKbmNNMYm9dY=";
```
Comment (kirillrdy) [inline pkgs/shells/fzf-git-sh/default.nix:26]:
```suggestion
    hash = "sha256-0KFxwx6wGaHXIK0FrAVkodWDMjc9zQjEKbmNNMYm9dY=";
```

**Comment by deejayem** [comment]
Why are there two unrelated commits (`maintainers: add mihaimaruseac` and `python3Packages.magika: init at 0.5.0`) showing up in the PR after the force push?

**Comment by becknik** [comment]
I accidentally rebased two commits prior to my initial branch commit. I didn't thought this might cause any issues (and still am confused, because this conflicts with my understanding of git), but here we are :|

**Comment by kirillrdy** [comment]
commit message should match version change eg `unstable-2022-09-30 -> 0-unstable-2024-03-17` and PR title should match commit message

**Review by deejayem — APPROVED** [review]
Built and tested on ` x86_64-darwin`

**Pull request merged**